### PR TITLE
CHWMBC: Test corrections

### PR DESCRIPTION
**Pull request by nkoeries**

### CODE REVIEW

File: primary-maternal-care/1.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 book: Primary Maternal Care
 title: 1. Antenatal care
 layout: chapter
-quiz-id: 131
+quiz-id: 90
```
Comment (arthurattwell) [inline primary-maternal-care/1.md]: @nkoeries Checking why this has changed. Did you figure out how to update the online tests? I'm impressed if you did, we hadn't got to that training yet :) Even then, the fact that this is an older test ID than the original (131) suggests you didn't create a new text but are using an old one. Can you fill me in on why you made this change?

File: primary-maternal-care/1B.md [hunk 2]
```diff
@@ -1,210 +1,878 @@
 ---
 book: Primary Maternal Care
-title: 1B. Examination of the abdomen in pregnancy
+title: 1. Antenatal care
```
Comment (arthurattwell) [inline primary-maternal-care/1B.md]: @nkoeries This looks like an old Skills chapter (1B) has been replaced with one of the updated chapters taken from Maternal Care (chapter 1). Was this a mistake or am I missing something?

File: primary-maternal-care/2.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 book: Primary Maternal Care
 title: 2. Assessment of fetal growth and condition during pregnancy
 layout: chapter
-quiz-id: 132
+quiz-id: 91
```
Comment (arthurattwell) [inline primary-maternal-care/2.md]: @nkoeries I think I've figured this out: these are the quiz IDs for Maternal Care? We must create new quizzes and not reuse the MC ones -- at at worst stick to told IDs (132 in this case). I'll have to show you how to do that. There should be basic info on the wiki.

The IDs refer to specific quizzes on quiz.bettercare.co.za, where we manage our online-quiz database.

File: primary-maternal-care/3.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 book: Primary Maternal Care
 title: 3. Hypertensive disorders of pregnancy
 layout: chapter
-quiz-id: 133
+quiz-id: 92
```
Comment (arthurattwell) [inline primary-maternal-care/3.md]: @nkoeries Again, here we must stick to 133 until we create new quizzes for PMC.

File: primary-maternal-care/4.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 book: Primary Maternal Care
 title: 4. Antepartum haemorrhage
 layout: chapter
-quiz-id: 134
+quiz-id: 93
```
Comment (arthurattwell) [inline primary-maternal-care/4.md]: @nkoeries Again, stick to 134 here till we have new tests for PMC.

File: primary-maternal-care/5.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 book: Primary Maternal Care
 title: 5. Preterm labour and preterm rupture of the membranes
 layout: chapter
-quiz-id: 135
+quiz-id: 94
```
Comment (arthurattwell) [inline primary-maternal-care/5.md]: @nkoeries Again, stick to 135 here till we have new tests for PMC.

File: primary-maternal-care/7.md [hunk 7]
```diff
@@ -1,14 +1,14 @@
 ---
 book: Primary Maternal Care
-title: 7. Medical problems during pregnancy and the puerperium
+title: . Medical problems during pregnancy, labour and the puerperium
```
Comment (arthurattwell) [inline primary-maternal-care/7.md]: @nkoeries Please add back the chapter number before the full stop at the start of the title here.

File: primary-maternal-care/7.md [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
 layout: chapter
-quiz-id: 137
+quiz-id: 102
```
Comment (arthurattwell) [inline primary-maternal-care/7.md]: @nkoeries Again, stick to 137 here till we have new tests for PMC.

File: primary-maternal-care/7.md (hunk 7 above)
Comment (arthurattwell) [reply primary-maternal-care/7.md]: @nkoeries Is there a chapter 6? I don't see one in the PR here.

**Review by arthurattwell — CHANGES_REQUESTED** [review]
@nkoeries I've left a few comments/requests for changes here. Can you make these and let me know when you have?

File: primary-maternal-care/7.md (hunk 7 above)
Comment (nkoeries) [reply primary-maternal-care/7.md]: The chapter 6 is there but it remains as is. Gerhard has made no changes to it.

File: primary-maternal-care/2.md (hunk 3 above)
Comment (nkoeries) [reply primary-maternal-care/2.md]: Got it. Will review Wiki again to get this right.

File: primary-maternal-care/1.md (hunk 1 above)
Comment (nkoeries) [reply primary-maternal-care/1.md]: I have not figured out how to change the online tests. What has happened is that Gerhard and Dave have updated the PMC chapters with the new MC. I began migrating the content including the test changes (clearing I was jumping the gun).

File: primary-maternal-care/3.md (hunk 4 above)
Comment (nkoeries) [reply primary-maternal-care/3.md]: @arthurattwell noted. I will make the changes.

File: primary-maternal-care/4.md (hunk 5 above)
Comment (nkoeries) [reply primary-maternal-care/4.md]: @arthurattwell noted. I will make the changes.

File: primary-maternal-care/7.md (hunk 8 above)
Comment (nkoeries) [reply primary-maternal-care/7.md]: @arthurattwell noted. I will make the changes.

File: primary-maternal-care/7.md (hunk 7 above)
Comment (nkoeries) [reply primary-maternal-care/7.md]: @arthurattwell Chapter number added back.

File: primary-maternal-care/1B.md (hunk 2 above)
Comment (nkoeries) [reply primary-maternal-care/1B.md]: @arthurattwell The content from Maternal Care replaces older material from Primary Maternal Care.

File: primary-maternal-care/1.md (hunk 1 above)
Comment (nkoeries) [reply primary-maternal-care/1.md]: @arthurattwell I will revert all the test IDs bearing in mind that the content has been changed.

File: primary-maternal-care/1B.md (hunk 2 above)
Comment (arthurattwell) [reply primary-maternal-care/1B.md]: @nkoeries What I mean is that you seem to have pasted the content for chapter 1 over Skills chapter 1B here.

Comment (nkoeries) [reply primary-maternal-care/1B.md]: @arthurattwell I see my mistake. I've corrected it. Trying to sync it now.

**Review by arthurattwell — APPROVED** [review]

**Pull request merged**